s3_to_redshift.go

```diff
@@ -113,6 +114,12 @@ func runCopy(db *redshift.Redshift, inputConf s3filepath.S3File, inputTable reds
 	return nil
 }
 
+func init() {
+	if *inputRegion != "" {
+		awsRegion = *inputRegion
```

Comment:
If we set this via command line argument, we should remove the environment variable here - otherwise it's confusing to a user which one is actually controlling the behavior